20210923/MINUTES.md

```diff
@@ -0,0 +1,87 @@
+# Minutes from the Sardana Follow-up Meeting - 2021/09/23
+
+To be held on Thursday 2021/09/23 at 14:00
+
+Participants:
+- MAXIV: Johan, Henrik, Vincent
+- ALBA: Zbignew, Marc, Saida
+- DESY: Teresa
+- Max Born: Daniel
+
+## Agenda
+
+0. Zibi presented two students (Marc and Saida) working at ALBA, on some sardana configuration related projects.
+
+1. Presentation by Teresa of the Sardana configuration tool used at DESY.
+  - The tool is called "sardana-aio" (all-in-one)
+  - Based on an XML file, listing everything that should go into the sardana config, 
+    + controllers 
+    + properties
+    + measurement groups
+    + default values (can be done with custom python code)
+  - Validates that Tango devices are running and not in FAULT
+  - A GUI is developed, and being taken into production
+
+2. Urgent user problems/issues - Round table
+   - MAX IV
+     - [X] Ctrl-C crash at MAX IV related to reading State. Henrik made a MR: https://github.com/sardana-org/sardana/pull/1695
+     - [ ] Issue at MAX IV related to ADLINK controller from ALBA, related to the State. Seems this is only in use at some specific place at ALBA. Zibi will check with them.
+   - ALBA
+     - [ ] Exotic problem with stateless subscriptions in PyTango that can cause issues in "fast dummy motor controller". Happens when taurus tries to subscribe but fails. Can be caused by changing event config. No decision on how to proceed.
+     - [ ] `self.<macro name>` exceptions are uninformative.
+     - [ ] Migration of final sardana 2 beamlines -> 3. Scans using two different pools together experience hangs. Perhaps some cases are related to state attribute polling. 
```

Comment:
```suggestion
     - Migration of final sardana 2 beamlines -> 3. 
     - [ ] Scans using two different pools together experience hangs. Perhaps some cases are related to state attribute polling. 
```
I would keep it as two separe points. And about migration, there is no action to be taken.